prune.go

```diff
@@ -134,8 +134,9 @@ func maybePrune(db *mgo.Database, txnsName string, pruneOpts PruneOptions) error
 	}
 
 	stats, err := CleanAndPrune(CleanAndPruneArgs{
-		Txns:      txns,
-		TxnsCount: txnsCount,
+		Txns:       txns,
+		TxnsCount:  txnsCount,
+		NewestTime: pruneOpts.NewestTime,
```

Comment:
I kinda like the idea of maybePrune using the lastPrune.Started for NewestTime although it end up being quite old for systems where txn growth is slow. We'd need to think about the impact of that.